src/agentscope/service/file/common.py

```diff
@@ -260,3 +272,92 @@ def get_current_directory() -> ServiceResponse:
             status=ServiceExecStatus.ERROR,
             content=error_message,
         )
+
+
+def write_file(
```

Comment:
Honestly, the `write_file` function is not for a general case. @ZiTao-Li @rayrayraykk What do you think?

Reply:
New service functions now in example.